tric-dt-book/about/book.md

```diff
@@ -0,0 +1,21 @@
+# About this Book
+
+```{warning} Coming Soon!
+Please note these project pages are a work-in-progress. Content will be added on an ongoing basis.
+```
```

Comment:
I wonder if we want to add some of these warning to the pages you've added too @cassgvp ? Might be good with a headsup for visitors!